asmcli/asmcli.sh

```diff
@@ -1272,8 +1322,13 @@ set_up_local_workspace() {
   fi
   pushd "$OUTPUT_DIR" > /dev/null
 
-  KUBECONFIG="asm_kubeconfig"
-  export KUBECONFIG
+  # If KUBECONFIG file is supplied, keep using that.
+  if [[ "${KUBECONFIG_SUPPLIED}" -eq 0 ]]; then
+    KUBECONFIG="asm_kubeconfig"
+    export KUBECONFIG
```

Comment:
I feel like we should export it regardless to make sure that children processes pick up the right file.

Reply:
makes sense, thanks.